### PR TITLE
Refactoring/tao 5104/remove globals from generis

### PR DESCRIPTION
**Pull request by ionutpad**
Depends on https://github.com/oat-sa/extension-tao-delivery-rdf/pull/156

### CODE REVIEW

File: helpers/DeliveryHelper.php [hunk 1]
```diff
@@ -107,12 +108,12 @@ public static function buildDeliveryData($delivery, $executions)
         }
         
         $deliveryProps = array(
-            new \core_kernel_classes_Property(TAO_DELIVERY_START_PROP),
-            new \core_kernel_classes_Property(TAO_DELIVERY_END_PROP),
+            new \core_kernel_classes_Property(DeliveryContainerService::PROPERTY_START),
```
Comment (zagovorichev) [inline helpers/DeliveryHelper.php]: I think that this wrong place for `PROPERTY_START`, it should be here `DeliveryAssemblyService`

**Comment by zagovorichev** [comment]
@jbout Could you check that constants have correct site location, please?

File: helpers/DeliveryHelper.php (hunk 1 above)
Comment (ionutpad) [reply helpers/DeliveryHelper.php]: The constant defined in the DeliveryAssemblyService are constants related to Assembled Delivery based on this fact I have separate them.

Comment (zagovorichev) [reply helpers/DeliveryHelper.php]: properties of the delivery should be in the one place
confirmed by @jbout

Comment (jbout) [reply helpers/DeliveryHelper.php]: these are properties related to the assembled delivery, they describe the time window during which the tt is allowed to start the test

**Comment by zagovorichev** [comment]
waiting for https://github.com/oat-sa/extension-tao-delivery-rdf/pull/156

**Pull request merged**